### DISCUSSION TITLE
How to visualize onnx .pb

### DISCUSSION THREAD

**Discussion by daydayfun** [onnx/onnx#2852] [Q&A]
I try to visualize onnx .pb,  I change it to human reading pbtxt, but  pbtxt is  still hard to read.
1. Can you help me find some tools to visualize onnx .pb input  (Like Netron, but it can not save the visualizztion graph to a pdf or svg)
2. If I run with resnet50.pb, and the accuracy descends , How to debug?

**Comment by jcwchen** [comment]
Hi @daydayfun, 
For 1, at least for the browser version of Netron, you can export as pdf or svg by the button in the top-left corner.

**Comment by daydayfun** [comment]
Thanks, the browser version of Netron is inconvenient  when i need to visualize  lots of networks. I want to find a tool, i write a python script to change the .pb to .dot, but the result is not good...

**Comment by prasanthpul** [comment]
@lutzroeder to comment on Netron

**Comment by lutzroeder** [comment]
See lutzroeder/netron#162. Since it requires a browser head the automation scenario isn't a priority.

Wondering if this script might help: https://github.com/onnx/tutorials/blob/master/tutorials/VisualizingAModel.md
